packages/components/src/components/Select/SelectMenu.tsx

```diff
@@ -0,0 +1,81 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import * as RadixPopover from '@radix-ui/react-popover';
+import { Slot as RadixSlot } from '@radix-ui/react-slot';
+import { type UseComboboxPropGetters, type UseSelectPropGetters } from 'downshift';
+import { isValidElement, type ForwardedRef, type ReactElement, type ReactNode } from 'react';
+
+import { type SelectItemProps } from './SelectItem';
+import styles from './styles/select.module.scss';
+import { getSelectOptionValue, recursiveMap } from './utils';
+
+export type SelectMenuProps = {
+    /**
+     * The index of the highlighted item in the menu.
+     */
+    highlightedIndex: number;
```

Comment:
Most of the props are internal right? (except `children`)
Can you mark with `@internal` in JSDoc?

Reply:
this whole component is internal, so maybe we remove the jsdoc alltogether?